### ISSUE TITLE
Poll Sample Does Not Work With Screen Readers

### ISSUE THREAD

**Issue by kmh287** [ampproject/amp-by-example#814]
Steps:
1. Enable a screen reader (like ChromeVox)
2. Navigate to https://ampbyexample.com/samples_templates/poll/preview/
3. Use TAB to focus on the first element in the radio group
4. Use Arrow keys to navigate between options

Result:
Form submits immediately upon changing options

Expected:
Form should allow users with screen readers to select an option, then provide a means to submit.

**Comment by sebastianbenz** [comment]
@kmh287 thanks! 

@kul3r4 can you take a look (after I/O)?
